src/TropicalGeometry/linear_space.jl

```diff
@@ -208,10 +209,25 @@ function tropical_linear_space(k::Int, n::Int, plueckerVector::Vector, nu::Union
 end
 
 
+# helper function that takes a matrix (tropical or algebraic)
+# and returns the non-zero minors and their indices
+function compute_pluecker_indices_and_vector(A::MatElem)
+    n = ncols(A)
+    k = nrows(A)
+    @req n>=k "matrix for Pluecker vector cannot have more rows than columns"
+    plueckerIndices = AbstractAlgebra.combinations(1:n,k)
+    plueckerVector = AbstractAlgebra.minors(A,k)
+    nonZeroIndices = findall(!iszero,plueckerVector)
+    plueckerIndices = plueckerIndices[nonZeroIndices]
+    plueckerVector = plueckerVector[nonZeroIndices]
+    return plueckerIndices, plueckerVector
+end
+
+
 @doc raw"""
     tropical_linear_space(A::MatElem{<:TropicalSemiringElem}; weighted_polyhedral_complex_only::Bool=false)
 
-Return a tropical linear space whose Pluecker vector are the tropical minors of `A`.  If `weighted_polyhedral_complex==true`, will not cache any extra information.
+Return a tropical linear space whose Pluecker vector are the tropical minors of `A`.  Assumes that `ncols(A)>=nrows(A)`.  If `weighted_polyhedral_complex==true`, will not cache any extra information.
```

Comment:
You could `@req ncols(A)>=nrows(A)`, it is probably not very expensive.

Reply:
Will be done!